lib/project_types/theme/commands/push.rb

```diff
@@ -27,6 +28,8 @@ def call(args, _name)
 
         theme = if (theme_id = options.flags[:theme_id])
           ShopifyCLI::Theme::Theme.new(@ctx, root: root, id: theme_id)
+        elsif options.flags[:live]
+          ShopifyCLI::Theme::Theme.live(@ctx, root: root)
```

Comment:
Do you think `--live` should imply `--allow-live`?

I'm drawn on this... but if you don't you'd have to `shopify theme push --live --allow-live` all the time. Seems repetitive.

Reply:
@macournoyer, I totally agree! I wondered the same thing, but I was afraid of making the live theme update too easy.

Still, the "live" word already warns users that they are performing a "live" operation. Thus, I've applied the changes in a new commit. Thanks for the suggestion! 🚀